lib/collective-sections.js

```diff
@@ -193,7 +193,7 @@ const getSectionsToRemoveForUser = (collective, isAdmin, isHostAdmin) => {
     // If there's no connected accounts, there's no benefit in enabling the section as it will return null anyway
     toRemove.add(Sections.CONNECTED_COLLECTIVES);
   }
-  if (!collective.hasLongDescription && !collective.longDescription && !isAdmin && !isHostAdmin) {
+  if (!collective.hasLongDescription && !collective.longDescription && !isAdmin) {
```

Comment:
@Betree : I think we can simplify this to; 

```suggestion
  if (!collective.hasLongDescription && !isAdmin) {
```

since, https://github.com/opencollective/opencollective-api/blob/347722b66d4b8b235b1688c9726c9db05a656d26/server/graphql/v1/CollectiveInterface.js#L966 isn't? 🤔

Reply:
It's in case the graphql query fetches `longDescription` but not `hasLongDescription`. It's actually the case for the collective page query: https://github.com/opencollective/opencollective-frontend/blob/30193763a062d5071501d6be908ce267e0a6cf22/components/collective-page/graphql/queries.js#L7

`hasLongDescription` is a way to fetch just this info in case we don't need the full HTML content, but using either one or another should be valid.

Reply:
Ah, thanks for the explanation. 😄